app/main/constants/config-const.js

```diff
@@ -6,7 +6,7 @@ angular.module('main')
   ENV: {
     /*inject-env*/
     'SERVER_URL': 'http://sample-env.qtbvngb2iz.us-west-2.elasticbeanstalk.com/',
```

Comment:
No, what i'm saying is change the name of AWS server's url in AWS itself. So that it is readable.